apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-advanced/dot-template-advanced.component.ts

```diff
@@ -9,6 +18,17 @@ import { DotTemplateItem } from '../store/dot-template.store';
 import { DotPortletToolbarActions } from '@models/dot-portlet-toolbar.model/dot-portlet-toolbar-actions.model';
 import { DotMessageService } from '@services/dot-message/dot-messages.service';
 
+interface operation {
```

Comment:
```suggestion
interface Operation {
```

This also needs better naming, Operation is too general.